components/aws/lambda-python/outputs.tf

```
@@ -1,3 +1,9 @@
 output "build_temp_dir" {
   value = local.temp_build_folder
 }
+output "function_ids" {
+  value = {
+    for name in local.function_names :
+    name => aws_lambda_function.python_lambda[name].id
```

Hi @aaronsteers , this needs to be arn instead of id to correctly feed into my step function and IAM roles - will this break something else?
```suggestion
    name => aws_lambda_function.python_lambda[name].arn
```
Separately, I realise one of my lambda functions needs extra permissions to perform a SageMaker action. Is it worth feeding my Step Functions IAM role for the functions?